x/emissions/types/keys.go

```diff
@@ -49,7 +49,8 @@ var (
 	InferenceScoresKey             = collections.NewPrefix(38)
 	ForecastScoresKey              = collections.NewPrefix(39)
 	ReputerListeningCoefficientKey = collections.NewPrefix(40)
-	FeeRevenueEpochKey             = collections.NewPrefix(41)
-	TopicFeeRevenueKey             = collections.NewPrefix(42)
-	PreviousTopicWeightKey         = collections.NewPrefix(43)
+	AverageRewardKey               = collections.NewPrefix(41)
```

Comment:
There are a few actors for which an avg reward can be calculated. this is for workers tho, so should be specified

Reply:
fixed this myself